### PR TITLE
2689 - MaterialUI - Buttons

### PR DESCRIPTION
**Pull request by LunAndrey**

### CODE REVIEW

File: jdi-light-material-ui/src/main/java/com/epam/jdi/light/material/elements/inputs/Button.java [hunk 1]
```diff
@@ -91,19 +43,4 @@ public boolean isDisabled() {
     public String getText() {
         return core().getText();
```
Comment (evasileva-epam) [inline jdi-light-material-ui/src/main/java/com/epam/jdi/light/material/elements/inputs/Button.java:44]: Do you really need all methods in this class?

File: jdi-light-material-ui/src/main/java/com/epam/jdi/light/material/asserts/inputs/ButtonAssert.java [hunk 2]
```diff
@@ -45,24 +45,6 @@ public ButtonAssert notVisible() {
         return this;
     }
```
Comment (evasileva-epam) [inline jdi-light-material-ui/src/main/java/com/epam/jdi/light/material/asserts/inputs/ButtonAssert.java:47]: Do you really need all methods in this class?

File: jdi-light-material-ui-tests/src/main/java/io/github/com/pages/inputs/ButtonPage.java [hunk 3]
```diff
@@ -3,37 +3,34 @@
 import com.epam.jdi.light.elements.composite.WebPage;
 import com.epam.jdi.light.elements.pageobjects.annotations.locators.UI;
 import com.epam.jdi.light.material.annotations.*;
```
Comment (evasileva-epam) [inline jdi-light-material-ui-tests/src/main/java/io/github/com/pages/inputs/ButtonPage.java:5]: Please delete grey import

File: jdi-light-material-ui-tests/src/main/java/io/github/com/pages/inputs/ButtonPage.java [hunk 4]
```diff
@@ -3,37 +3,34 @@
...
-import com.epam.jdi.light.material.elements.inputs.Button;
+import com.epam.jdi.light.ui.html.elements.common.Button;
```
Comment (evasileva-epam) [inline jdi-light-material-ui-tests/src/main/java/io/github/com/pages/inputs/ButtonPage.java:6]: Do you really don't need material Button import?

File: jdi-light-material-ui-tests/src/test/java/io/github/epam/material/tests/inputs/ButtonTests.java [hunk 5]
```diff
@@ -1,179 +1,152 @@
...
-        button.getButtonByGroupIndex("contained",5).click();
-        button.getButtonByGroupIndex("contained",5).is().hasClass("MuiButton-contained");
-        button.getButtonByGroupIndex("contained",5).is().text("LINK");
-        button.getButtonByGroupIndex("contained",5).is().hasClass("MuiButton-containedPrimary");
-        checkContainedField.is().text("Last click: Link");
-        button.getButtonByGroupIndex("contained",5).is().notVisible();
+        containedButtons.get(1).click();
+        containedButtons.get(1).is().text("DEFAULT");
+        containedButtons.get(6).is().text("Last click: Default");
+
```
Comment (evasileva-epam) [inline jdi-light-material-ui-tests/src/test/java/io/github/epam/material/tests/inputs/ButtonTests.java:23]: Please delete extra line

**Review by evasileva-epam — CHANGES_REQUESTED** [review]

File: jdi-light-material-ui/src/main/java/com/epam/jdi/light/material/elements/inputs/Button.java (hunk 1 above)
Comment (LunAndrey) [reply jdi-light-material-ui/src/main/java/com/epam/jdi/light/material/elements/inputs/Button.java:44]: this method uses in another material tests

File: jdi-light-material-ui/src/main/java/com/epam/jdi/light/material/asserts/inputs/ButtonAssert.java (hunk 2 above)
Comment (LunAndrey) [reply jdi-light-material-ui/src/main/java/com/epam/jdi/light/material/asserts/inputs/ButtonAssert.java:47]: only hasClass method can be deleted, other methods from this class uses other tests

File: jdi-light-material-ui-tests/src/main/java/io/github/com/pages/inputs/ButtonPage.java (hunk 4 above)
Comment (LunAndrey) [reply jdi-light-material-ui-tests/src/main/java/io/github/com/pages/inputs/ButtonPage.java:6]: I used only html Button import, and don't need to use Button inport form material

**Pull request merged**